pkg/lvm/lvm_util.go

```diff
@@ -130,6 +130,11 @@ func CreateVolume(vol *apis.LVMVolume) error {
 
 // DestroyVolume deletes the lvm volume
 func DestroyVolume(vol *apis.LVMVolume) error {
+	if vol.Spec.VolGroup == "" {
```

Comment:
can it happen?

Reply:
May be in one corner scenario where deletion of lvmvol happens just after creation but before provisioning. e.g our creation workflow gets failed due to some reason & gets retried after some time but deletion event gets triggered in between.